services/src/server.rs

```diff
@@ -73,16 +78,21 @@ pub async fn start_server(
             .await
         }
         Backend::Postgres => {
-            eprintln!("Using Postgres backend"); // TODO: log
-            let ctx = PostgresContext::new(
-                tokio_postgres::config::Config::from_str(
-                    &get_config_element::<config::Postgres>()?.config_string,
-                )?,
-                NoTls,
-            )
-            .await?;
-
-            start(shutdown_rx, static_files_dir, bind_address, ctx).await
+            #[cfg(feature = "postgres")]
```

Comment:
Would it be possible to just opt-in/out the enum variant at all?

Reply:
I would rather report an error if the user configured the backend to use postgres than silently use the in memory variant.